xcell/tests/test_cls.py

```diff
@@ -294,6 +298,13 @@ def test_symmetric():
     shutil.rmtree(tmpdir1)
 
 
+def test_unsupported_quantity():
+    data = get_config(dtype0='generic')
+    with pytest.raises(ValueError):
```

Comment:
If you agree with me on using `NotImplementedError`, update this

Reply:
Done